docs/developer/watcher-local-test.md

```diff
@@ -1,142 +1,167 @@
-# Test watcher locally in two-cluster-setup using K3D
+# Test watcher locally in two-cluster-setup using k3d
 
-### Create KCP and SKR clusters
-1. Run the following command to create a local control-plane (KCP) cluster:
-```shell
-k3d cluster create kcp-local --port 9080:80@loadbalancer \
---k3s-arg '--no-deploy=traefik@server:0'
-```
-2. Run the following command to create a local kyma-runtime (SKR) cluster:
-```shell
-k3d cluster create skr-local
-```
-### Switch context to the KCP cluster config
-Run the following command to switch `kubectl` config context to prepare for running the subsequent commands:
-```shell
-kubectl config use-context k3d-kcp-local
-```
-### Install Istio and Manifest CRD
-1. Run the following command to install `istioctl` using `brew`:
-```shell
-brew install istioctl
-```
-2. Run the following command to install Istio and Manifest CRD on the KCP cluster:
-```shell
-istioctl install --set profile=demo -y && \
-kubectl apply -f https://raw.githubusercontent.com/kyma-project/module-manager/main/config/crd/bases/operator.kyma-project.io_manifests.yaml
-```
+### KCP cluster setup
+   
+1. Create a local control-plane (KCP) cluster:
+    ```shell
+    k3d cluster create kcp-local --port 9080:80@loadbalancer \
+    --registry-create k3d-registry.localhost:0.0.0.0:5111 \
+    --k3s-arg '--no-deploy=traefik@server:0'
+    ```
+2. Deploy lifecycle-manager on the cluster:
```

Comment:
This command complains about missing module-manager CRDs. Could you place it after the step where we are installing module-manager CRDs.